client/src/panels/internal_services/isc.js

```diff
@@ -28,44 +13,43 @@ import {
 const { Gov, Tag } = Subject;
 const { std_years } = years;
 const { GraphLegend } = declarative_charts;
-
 const { text_maker, TM } = create_text_maker_component(text);
 
-
 export const declare_internal_services_panel = () => declare_panel({
   panel_key: "internal_services",
   levels: ["dept"],
-  panel_config_func: (level, panel_key) => ({
-    depends_on: ['programFtes', "programSpending"],
+  panel_config_func: () => ({
+    depends_on: ['programFtes'],
     title: "internal_service_panel_title",
-    calculate(subject,info){
+    calculate(subject){
       const { programFtes } = this.tables;
   
       const isc_crsos = _.filter(subject.crsos, "is_internal_service");
       if(_.isEmpty(isc_crsos)){
-        //org has no isc programs 
-        return false;
+        return false; //org has no isc programs 
```

Comment:
Ok, the problem with FCAC is that it *does* technically have an internal services CR (as of 2018-19), but it doesn't actually have any internal service spending or FTEs (because we're not done 2018-19 yet). Partly that will fix itself after public accounts are out, but the 5-year graph will be a little weird still (because it will have 0s for the four previous years).

What you will need to do to calculate "first active year" and "last active year" the way the people tables do. You can see how it's done for employee age in lines 130-148 of `orgEmployeeAvgAge.js`.